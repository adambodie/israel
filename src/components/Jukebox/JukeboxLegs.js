import React from 'react';

const JukeboxLegs = () => {
	return (
		<div className='legs'>
			<div className='leg-bottom'></div>
			<div className='leg'></div>
			<div className='leg'></div>
		</div>
	);
};

export default JukeboxLegs;
