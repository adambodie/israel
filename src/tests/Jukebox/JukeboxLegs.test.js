import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import JukeboxLegs from '../../components/Jukebox/JukeboxLegs';
import {
	render,
	cleanup
  } from 'react-testing-library'
  
  afterEach(cleanup)

it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(<JukeboxLegs />, div);
	ReactDOM.unmountComponentAtNode(div);
});

it('renders correctly', () => {
	const tree = renderer.create(<JukeboxLegs />).toJSON();
	expect(tree).toMatchSnapshot();
});