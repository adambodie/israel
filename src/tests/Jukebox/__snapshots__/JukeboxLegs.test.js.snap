// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="legs"
>
  <div
    className="leg-bottom"
  />
  <div
    className="leg"
  />
  <div
    className="leg"
  />
</div>
`;
